test/services/conversion/TransitAccompanyingDocumentConverterSpec.scala

```diff
@@ -15,17 +15,25 @@
  */
 
```

Comment:
Similar comments to TransitAccompanyingDocumentConversionServiceSpec.scala regarding the data for principal/consignor/consignee eori/name and address fields

Reply:
The TransitAccompanyingDocumentConverter is more about testing the reference data is being set properly within the view model. The updates to TransitAccompanyingDocumentConversionServiceSpec should cover your queries above.